Cargo.toml

```diff
@@ -18,3 +18,11 @@ resolver = "2"
 
 [profile.release]
 lto = true
+
+[profile.release-quick]
```

Comment:
Is it release-fast or release quick?

Reply:
Whoops, I couldn't decide on a name so I guess I used both. I think fast is better (less keys to type).